src/qgis_plugin_dev_tools/build/packaging.py

```
@@ -133,6 +135,30 @@ def copy_runtime_requirements(
                 )
 
 
+def copy_license(dev_tools_config: DevToolsConfig, build_directory_path: Path) -> None:
+    plugin_build_path = build_directory_path / dev_tools_config.plugin_package_name
+    target_license_file = plugin_build_path / "LICENSE"
+    if target_license_file.exists():
+        LOGGER.debug(f"Existing license file {target_license_file} found.")
+        return
+
+    licenses_to_test = [
+        dev_tools_config.license_file_path,
```

is this a bit dangerous, since a missing configured file will be silently skipped if there is a candidate present?

should this be the first case, if this is not none, try to copy that? maybe warning in case it overwrites a license already in the build dir? now it skips a configured file on debug level logging, if there is a license already even if you configure something explicitly?

only then check if there is one already, and if not, try to copy the candidate if not none, else log warning?

---

Yea, there is this possibility. I'll change it.